src/samplers/ratelimiting_sampler.js

```diff
@@ -32,7 +32,7 @@ export default class RateLimitingSampler {
         }
 
         this._maxTracesPerSecond = maxTracesPerSecond;
-        this._rateLimiter = new RateLimiter(maxTracesPerSecond);
+        this._rateLimiter = new RateLimiter(maxTracesPerSecond, 2.0);
```

Comment:
The max balance should equal the checkCredit value (1.0) in this case. But I set it to 2 so that all existing tests can work with all the edge cases which unfortunately 1.0 doesn't hit. The only difference in practice is that we buffer up to 2 traces which isn't toooooo bad.